py_gtfs_rt_ingestion/lib/convert_gtfs_rt.py

```diff
@@ -115,12 +122,6 @@ def process_files(self) -> Iterable[pyarrow.table]:
             try:
                 timestamp, new_data = self.gz_to_pyarrow(file)
 
-                # skip files that have been generated after the start of the
-                # current hour. don't add them to archive or error so that they
-                # are picked up next go around.
-                if timestamp >= self.start_of_hour:
-                    break
-
```

Comment:
Moved this to after our yield in the hour break. We should only be yielding on hour breaks and not ever yielding a table that only contains a partial hour